### PR TITLE
[ops, model] refactor: shorten loss-wrapper return to (loss, logits, fused_linear_aux)

### PR DESCRIPTION
**Pull request by Luosuu**
## Summary

Follow-up to #771 implementing the second half of [Coach257's design proposal](https://github.com/ByteDance-Seed/VeOmni/pull/771#issuecomment-4495792930): now that `fused_linear_aux` is the canonical home for per-token tensors, the loss wrapper return shape can collapse from a 7-tuple to a 3-tuple.

**Before** (loss wrapper returned 5 trailing `None`s on the loss path; each patchgen `forward` unpacked 7 locals + wrapped them via `FusedLinearAuxOutput.from_loss_slots(...)` at the output ctor):
```python
loss, _, log_probs, entropy, distillation_losses, student_mass, teacher_mass = self.loss_function(...)
return CausalLMOutputWithLogProbs(
    loss=loss, logits=logits,
    fused_linear_aux=FusedLinearAuxOutput.from_loss_slots(
        log_probs=log_probs, entropy=entropy,
        distillation_losses=distillation_losses,
        student_mass=student_mass, teacher_mass=teacher_mass,
    ),
    ...
)
```

**After** (wrapper builds the payload internally; every model `forward` is a one-liner regardless of which sub-path ran):
```python
loss, logits, fused_linear_aux = self.loss_function(...)
return CausalLMOutputWithLogProbs(loss=loss, logits=logits, fused_linear_aux=fused_linear_aux, ...)
```

## Changes

- `veomni/ops/kernels/cross_entropy/__init__.py`:
  - `ForCausalLMLoss` and `_chunk_loss_dispatch` return `(loss, logits, FusedLinearAuxOutput | None)`. They now instantiate `FusedLinearAuxOutput` directly on the log-probs and top-k distillation branches.
  - `ForSequenceClassificationLoss` returns `(loss, logits, None)` (no per-token tensors on seq-cls).
- `veomni/models/transformers/*/_patch_gen_config.py` (17 configs): 7-tuple unpack → 3-tuple, 5 local-var inits collapsed to one, output ctor takes `fused_linear_aux=fused_linear_aux` directly, dead `FusedLinearAuxOutput` imports removed. Regenerated via `make patchgen` + `check_patchgen --fix`.
- `tests/ops/test_seqcls_loss.py`: 4 unpack sites bumped from 7-tuple to 3-tuple.

## Why

Adding a new per-token tensor (value head, reward, alt-KL, etc.) now extends `FusedLinearAuxOutput` in **one place** — no edits to the wrapper return shape and no edits to any patchgen `forward`. The previous flat-tuple shape required touching every call site for each new metric.

## Test plan

- [x] Worker pytest (H100): 33 passed, 4 skipped, 0 failed on commit `51d8344`. Covers `test_chunk_topk_distill.py` (10 tests incl. memory-savings), `test_chunk_logprobs.py`, `test_seqcls_loss.py` (6 tests post-bump), `test_return_log_probs_e2e.py` (10 parametrizations).
- [x] `make patchgen` + `check_patchgen --fix` clean.
- [x] `make quality` clean.
- [x] Smoke-import of patched modeling files (qwen2, qwen3, qwen3_5, qwen3_5_moe, qwen3_vl, deepseek_v3) — confirms HF `@auto_docstring` is happy with the unchanged `*WithLogProbs` docstrings.
- [ ] CI on this PR.

### CODE REVIEW

**Review by TimYangst — APPROVED** [review]
Overal looks good to me, however, please take look on the comments caught by the gemini. And please resolve the conflict before merge the PR.

File: veomni/models/transformers/qwen3_5_moe/qwen3_5_moe_gpu_patch_gen_config.py [hunk 1]
```diff
@@ -874,9 +870,9 @@ def qwen3_5_moe_forcausallm_forward_patched(
         else:
             logits = self.lm_head(hidden_states)
             # Modification: VeOmni's patched `loss_function` (via LOSS_MAPPING)
-            # returns (loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass); unpack to match the
+            # returns (loss, logits, fused_linear_aux); unpack to match the
             # OpSlot branch above.
-            loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass = self.loss_function(
+            loss, logits, fused_linear_aux = self.loss_function(
```
[the remark this answers is not in the document]
Comment (Luosuu) [reply veomni/models/transformers/qwen3_5_moe/qwen3_5_moe_gpu_patch_gen_config.py]: Fixed in e744253 — switched to `loss, _, fused_linear_aux = self.loss_function(...)` and added the standard `if fused_linear_aux is not None: logits = None` clear branch (matching the qwen3 / qwen2 / llama configs). Also applied the same fix to the two sibling MoE configs you did not flag (qwen3_vl_moe and qwen3_omni_moe) since they had the identical pattern. 6 sites across 5 files; generated files regenerated.

File: veomni/models/transformers/qwen3_5_moe/qwen3_5_moe_gpu_patch_gen_config.py [hunk 2]
```diff
@@ -988,9 +974,9 @@ def qwen3_5_moe_forconditional_generation_forward_patched(
...
                 logits=logits,
```
[the remark this answers is not in the document]
Comment (Luosuu) [reply veomni/models/transformers/qwen3_5_moe/qwen3_5_moe_gpu_patch_gen_config.py:990]: Fixed in e744253 — switched to `loss, _, fused_linear_aux = self.loss_function(...)` and added the standard `if fused_linear_aux is not None: logits = None` clear branch (matching the qwen3 / qwen2 / llama configs). Also applied the same fix to the two sibling MoE configs you did not flag (qwen3_vl_moe and qwen3_omni_moe) since they had the identical pattern. 6 sites across 5 files; generated files regenerated.

File: veomni/models/transformers/qwen3_moe/qwen3_moe_gpu_patch_gen_config.py [hunk 3]
```diff
@@ -338,9 +334,9 @@ def qwen3_moe_forcausallm_forward_patched(
         else:
             logits = self.lm_head(hidden_states)
             # Modification: VeOmni's patched `loss_function` (via LOSS_MAPPING)
-            # returns (loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass); unpack to match the
+            # returns (loss, logits, fused_linear_aux); unpack to match the
             # OpSlot branch above.
-            loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass = self.loss_function(
+            loss, logits, fused_linear_aux = self.loss_function(
                 logits=logits,
```
[the remark this answers is not in the document]
Comment (Luosuu) [reply veomni/models/transformers/qwen3_moe/qwen3_moe_gpu_patch_gen_config.py:340]: Fixed in e744253 — switched to `loss, _, fused_linear_aux = self.loss_function(...)` and added the standard `if fused_linear_aux is not None: logits = None` clear branch (matching the qwen3 / qwen2 / llama configs). Also applied the same fix to the two sibling MoE configs you did not flag (qwen3_vl_moe and qwen3_omni_moe) since they had the identical pattern. 6 sites across 5 files; generated files regenerated.

File: veomni/models/transformers/qwen2_5_omni/qwen2_5_omni_gpu_patch_gen_config.py [hunk 4]
```diff
@@ -1193,9 +1189,9 @@ def qwen2_5_omni_thinker_forward_patched(
         else:
             logits = self.lm_head(hidden_states)
             # Modification: VeOmni's patched ``loss_function`` (via
-            # LOSS_MAPPING) returns ``(loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass)``;
+            # LOSS_MAPPING) returns ``(loss, logits, fused_linear_aux)``;
             # unpack to match the OpSlot branch above.
-            loss, logits, log_probs, entropy, distillation_losses, student_mass, teacher_mass = self.loss_function(
+            loss, logits, fused_linear_aux = self.loss_function(
```
[the remark this answers is not in the document]
Comment (Luosuu) [reply veomni/models/transformers/qwen2_5_omni/qwen2_5_omni_gpu_patch_gen_config.py]: Fixed in e744253 — switched to `loss, _, fused_linear_aux = self.loss_function(...)` and added the standard `if fused_linear_aux is not None: logits = None` clear branch (matching the qwen3 / qwen2 / llama configs). Also applied the same fix to the two sibling MoE configs you did not flag (qwen3_vl_moe and qwen3_omni_moe) since they had the identical pattern. 6 sites across 5 files; generated files regenerated.

**Review by TimYangst — APPROVED** [review]
